README.md

```diff
@@ -1,10 +1,10 @@
-# Office Addin Debugger Extension for Visual Studio Code
+# Microsoft Office Add-in Debugger for Visual Studio Code
 
-The "Office Addin Debugger" Extension for Visual Studio Code allows you to use F5 to debug your Office Add-in against the Edge runtime.
+The "Office Add-in Debugger" Extension for Visual Studio Code allows you to use F5 to debug your Office Add-in against the Edge runtime.

```

Comment:
```suggestion
The Microsoft Office Add-in Debugger Extension for Visual Studio Code allows you to use F5 to debug your Office Add-in against the Edge runtime.

```

Reply:
Yup I caught this too and pushed a commit already